src/utils/DropdownButton.vue

```diff
@@ -37,7 +37,7 @@
                     :data-cy="`dropdown-item-${item.title.toLowerCase()}`"
                     @click="selectItem(item)"
                 >
-                    {{ $t(item.title) }}
```

Comment:
The documentation on line 55-57 should be fixed to reflect that change.

Reply:
Done